plugins/postcss-cascade-layers/src/compound-selector-order.ts

```diff
@@ -0,0 +1,88 @@
+import selectorParser from 'postcss-selector-parser';
```

Comment:
This file and functions therein allow us to concatenate any selector with `:not()` into a non-broken selector.

This prevents issues that would arise from `.foo::before` + `:not()`